api/room/schema_query.py

```diff
@@ -115,6 +115,7 @@ class Query(graphene.ObjectType):
     """
     all_rooms = graphene.Field(
         PaginatedRooms,
+        room_labels=graphene.String(),
```

Comment:
Good work on the PR @TeamoreA,
Kindly add room label to the description on line `126`

Reply:
This has been added @jesseinit